### PR TITLE
Apply PHP 8.0 Syntax and constructor promotion

### PR DESCRIPTION
**Pull request by samsonasik**
Signed-off-by: Abdul Malik Ikhsan <samsonasik@gmail.com>

|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Since composer.json require php 8.0, php 8.0 syntax can constuctor promotion can be applied.

### CODE REVIEW

**Comment by samsonasik** [comment]
All green 🎉

File: src/TwigExtension.php [hunk 1]
```diff
@@ -15,33 +15,16 @@
...
-
-    private ?string $assetsUrl;
-
-    /** @var null|string|int */
-    private $assetsVersion;
-
-    private array $globals;
-
     /**
      * @param null|string|int $assetsVersion
```
Comment (Ocramius) [inline src/TwigExtension.php]: Can this now be a native type?

Comment (samsonasik) [reply src/TwigExtension.php]: ok, updated https://github.com/mezzio/mezzio-twigrenderer/pull/25/commits/c4278b9e583588fe5cdce37b47d172af887969ed

**Review by Ocramius — APPROVED** [review]
Thanks @samsonasik!

**Pull request merged**